core/src/integration/java/com/radixdlt/integration/steady_state/simulation/consensus/UniformLatencyTest.java

```diff
@@ -93,7 +95,13 @@ public void given_4_correct_bfts__then_should_pass_sanity_tests_over_1_minute()
     SimulationTest bftTest =
         SimulationTest.builder()
             .networkModules(NetworkOrdering.inOrder(), NetworkLatencies.fixed())
-            .numNodes(4)
+            .numPhysicalNodes(4)
+            .functionalNodeModule(
+                new FunctionalRadixNodeModule(
+                    false,
+                    SafetyRecoveryConfig.mocked(),
+                    ConsensusConfig.of(),
+                    LedgerConfig.mocked(4)))
```

Comment:
I'm not sure what this parameter means, perhaps `mocked` could do with a slightly better name?